lib/create-pull-request.js

```diff
@@ -80,10 +87,29 @@ async function createPullRequest(
           }
         }
 
+        // Text files can be changed through the .content key
+        if (isString(value)) {
```

Comment:
Can you elaborate why you don't check with `typeof value === 'string'`?

Reply:
All existing tests pass when checking with `typeof value === 'string'` instead

Reply:
TBH I never know how to test for sure if a var is a string in plain JS (I usually have _.isString from lodash). So I went [to Stack Overflow](https://stackoverflow.com/questions/4059147/check-if-a-variable-is-a-string-in-javascript) and this seemed like the most robust answer but I didn't dig any deeper than that...

So, in a nutshell, I'm fine with the `typeof` check